pydantic_ai_slim/pydantic_ai/tools.py

```diff
@@ -351,6 +351,9 @@ class ToolDefinition:
     description: str | None = None
     """The description of the tool."""
 
+    allow_parallel: bool = True
```

Comment:
Let's move this after `strict`

Reply:
I _think_ I like `sequential=True` better than `allow_parallel=False` -- what do you think?

Reply:
sounds good